CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+Unreleased
+----------
+- cmd/smith/app: renamed `ApiExtClient` field to `APIExtClient`
+- cmd/smith/app: renamed `ScClient` field to `SCClient`
```

Comment:
We don't provide any code compatibility guarantees so there is no need to call this out.

Reply:
Do we? The suggested deployment model for smith with plugins is depending directly on the source (and everything that's public in it). In a more general sense its probably good to start putting the effort into updating this when things change.

Reply:
Then we need to define what parts of exported code are considered public API. I also don't want us to slow down because "we cannot make breaking changes". That, of course, does not mean that we cannot start documenting them.

Reply:
It's go, everything that starts with an upper case letter is fair game. If that's too much we need some sort of versioned API pkg.

Reply:
That is the thing - I don't think we need to do anything here. We have 20 issues in this project. Each of them is more important than tracking breaking changes at this time (which is work). This is just my opinion, of course.